src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/Drawable.cs

```diff
@@ -84,6 +100,43 @@ public Vector4 BoundingBox
             }
         }
 
+        /// <summary>
+        /// The intersection with clip drawable is determined and only the resulting pixels from own drawable are rendered.
+        /// </summary>
+        /// <param name="clip">The clip drawable object.</param>
+        /// <returns>True when it's successful. False otherwise.</returns>
+        /// <exception cref="ArgumentNullException"> Thrown when drawable is null. </exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool ClipPath(Drawable clip)
+        {
+            if (clip == null)
+            {
+                throw new ArgumentNullException(nameof(clip));
+            }
+            bool ret = Interop.Drawable.SetClipPath(View.getCPtr(this), BaseHandle.getCPtr(clip));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        /// <summary>
+        /// The pixels of mask drawable and own drawable are blended according to MaskType.
+        /// </summary>
+        /// <param name="mask">The mask drawable object.</param>
+        /// <param name="type">The masking type.</param>
+        /// <returns>True when it's successful. False otherwise.</returns>
```

Comment:
For the consistency you do better to use exception to inform error instead returning bool

Reply:
I modified code by changing the return type and throwing an exception.